ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -293,26 +294,14 @@ const BaseVariantLocusListLabels = React.memo(({ locusListIntervalsByProject, fa
   if (!locusListIntervalsByProject || locusListIntervalsByProject.length < 1) {
     return null
   }
-  const { pos, end, genomeVersion, liftedOverPos, familyGuids = [] } = variant
+  const { familyGuids = [] } = variant
   const locusListIntervals = familyGuids.reduce((acc, familyGuid) => ([
     ...acc, ...(locusListIntervalsByProject[familiesByGuid[familyGuid].projectGuid] || [])]), [])
   if (locusListIntervals.length < 1) {
     return null
   }
-  const locusListGuids = locusListIntervals.filter((interval) => {
-    const variantPos = genomeVersion === interval.genomeVersion ? pos : liftedOverPos
-    if (!variantPos) {
-      return false
-    }
-    if ((variantPos >= interval.start) && (variantPos <= interval.end)) {
-      return true
-    }
-    if (end && !variant.endChrom) {
-      const variantPosEnd = variantPos + (end - pos)
-      return (variantPosEnd >= interval.start) && (variantPosEnd <= interval.end)
-    }
-    return false
-  }).map(({ locusListGuid }) => locusListGuid)
+  const locusListGuids = locusListIntervals.filter(variantIntervalOverlap(variant))
+    .map(({ locusListGuid }) => locusListGuid)
```

Comment:
It looks no simple way to make the `VariantLocusListLabels` reusable for `SpliceOutlierLabel`. So I just created a reusable variant and `Interval` overlapping checking function.